packages/graphql/pubspec.yaml

```diff
@@ -20,9 +20,10 @@ dependencies:
   websocket: ^0.0.5
   quiver: '>=2.0.0 <3.0.0'
 dev_dependencies:
-  pedantic: ^1.8.0+1
-  mockito: ^4.0.0
-  test: ^1.5.3
-  test_coverage: ^0.3.0+1
+  pedantic: ^1.9.0
```

Comment:
This might break with the next version of flutter which requires vs 1.7 if I am not mistaken. Can you test with beta and see if it works.

Reply:
sorry, just trying to figure out why the test_coverage is hanging

Reply:
I will look into it tomorrow and see if I can also figure out why.